packages/core/core-flows/src/draft-order/workflows/confirm-draft-order-edit.ts

```diff
@@ -134,31 +134,21 @@ export const confirmDraftOrderEditWorkflow = createWorkflow(
       throw_if_key_not_found: true,
     }).config({ name: "order-items-query" })
 
-    const { removedLineItemIds } = transform(
-      { orderItems, previousOrderItems: order.items },
-      (data) => {
-        const previousItemIds = (data.previousOrderItems || []).map(
-          ({ id }) => id
-        )
-        const currentItemIds = data.orderItems.items.map(({ id }) => id)
+    const { variants, items, toRemoveReservationLineItemIds } = transform(
```

Comment:
thought: Can we consolidate these transforms? I believe we have almost the exact same code in many places, and I fear we can easily end up updating one but not the others.

Reply:
Sounds good, I can open a separate ticket.